src/topic_table_model.h

```diff
@@ -25,12 +24,11 @@ class TopicTableModel : public QAbstractTableModel
     Q_OBJECT
 
 public:
-
     /**
      * @brief Constructor for the DDS sample data model.
      * @param[in] parent The parent for this data model.
      */
-    TopicTableModel(QTableView* parent, const QString& topicName);
+    TopicTableModel(QTableView *parent, const QString &topicName);
```

Comment:
This project doesn't have a style guideline yet, but please use a style as consistent with other files as possible. For this specific line, the `*` and `&` should be close to the types instead of the parameters.